### PR TITLE
Add default keybindings for sub-scale

### PR DESCRIPTION
**Pull request by soredake**
Read this before you submit this pull request:
https://github.com/mpv-player/mpv/blob/master/DOCS/contribute.md

Reading this link and following the rules will get your pull request reviewed
and merged faster. Nobody wants lazy pull requests.

### CODE REVIEW

**Comment by wm4** [comment]
Read the link in your post.

**Comment by soredake** [comment]
Better?

**Comment by wm4** [comment]
Better (doesn't match how the git history usually has it (without the /etc/) but the rules are pretty arbitrary anyway).
You still need to add this to the manpage and possibly other places that list key bindings.

**Comment by soredake** [comment]
Added changes to manpage.

**Comment by soredake** [comment]
@jeeb @sfan5 dunno whom to ping

File: DOCS/man/mpv.rst [hunk 1]
```diff
@@ -159,6 +159,9 @@ L
 Ctrl + and Ctrl -
     Adjust audio delay (A/V sync) by +/- 0.1 seconds.
 
+Shift+g and Shift+f
+    Adjust subtitle font size by +/- 0.1.
```
Comment (sfan5) [inline DOCS/man/mpv.rst:163]: Wouldn't "increase subtitle scaling by +/- 10%" be more correct?

Comment (soredake) [reply DOCS/man/mpv.rst]: Done

**Review by sfan5 — APPROVED** [review]

**Comment by soredake** [comment]
Thanks.

**Pull request merged**